parm/config/config.prep

```diff
@@ -9,6 +9,7 @@ echo "BEGIN: config.prep"
 . $EXPDIR/config.resources prep
 
 export DO_MAKEPREPBUFR="YES"   # if NO, will copy prepbufr from globaldump
+export cdate10=${PDY}${cyc}
```

Comment:
Who is needing this variable to be set in their env.?
Is production setting this in `ecf` of the obsprocs?
Just curious.

Reply:
We have to set `cdate10` for obsproc `scripts/exglobal_makeprepbufr.sh`:

```
cdate10=${cdate10:-`cut -c7-16 ncepdate`}
```

https://github.com/NOAA-EMC/obsproc/blob/develop/scripts/exglobal_makeprepbufr.sh#L78

I had them add the override this year so we can pass `cdate10` in for development runs. In ops the default (`ncepdate`) is used. The same update was made in the GFSv16 system.